src/Models/Classes/Errors.ts

```diff
@@ -56,7 +56,8 @@ class ComponentError extends Error implements IComponentError {
                 return i18n.t('errors.schemaValidationFailed.message');
             case ComponentErrorType.TimeSeriesDatabaseConnectionFetchFailed:
                 return i18n.t(
-                    'errors.timeSeriesDatabaseConnectionFetchFailed.message'
+                    'errors.timeSeriesDatabaseConnectionFetchFailed.message',
+                    { permission: 'Reader' }
```

Comment:
this should be localized no?

Reply:
I wanted to keep the permission name out of the translation since it might be a bad translation. In the portal right now, we can see the name of permissions translated if another language is selected but a very specific way. We can have a list of permanent translation section in our language files or another place like enums.

Reply:
yep. makes sense. Let's leave it as is since that's the permission name.